mavlink-bindgen/src/parser.rs

```diff
@@ -333,6 +337,48 @@ impl MavProfile {
             }
         }
     }
+
+    fn emit_mav_message_target_system_id(&self) -> TokenStream {
+        let arms: Vec<TokenStream> = self
+            .messages
+            .values()
+            .filter(|msg| msg.fields.iter().any(|f| f.name == "target_system"))
```

Comment:
Do we ever have more than one `target_system` field in a single message? If not, we should panic in that case.

cc @patrickelectric @joaoantoniocardoso

Reply:
Looks good to me. :)

Reply:
Alright, shall we merge this PR then? :slightly_smiling_face: